getting-started/kubernetes/self-managed-onprem/index.md

```diff
@@ -7,5 +7,8 @@ show_toc: false
 
 {{ page.description }}
 
+Depending on your needs, Calico can be dedicated to Kubernetes networking while connecting to existing
```

Comment:
It could just be me, but this sentence reads slightly strange to me.  What does it mean for Calico to be dedicated to Kubernetes networking?